front/app/utils/helperUtils.ts

```diff
@@ -223,3 +224,7 @@ export function hexToRGBA(hex: string, alpha: number) {
 export const classNames = (...classes: (string | undefined)[]) => {
   return classes.filter(Boolean).join(' ');
 };
+
+export function generateTempId() {
```

Comment:
Moved this to shared utils since it was imported all over the codebase, and the previous file it was in had a lot of imports of ESRI and stuff which was breaking Jest